core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -254,10 +254,32 @@ export class CustomizeInteractionModalComponent
   }
 
   isSaveInteractionButtonEnabled(): boolean {
-    return !!(
-      this.hasCustomizationArgs &&
-      this.stateInteractionIdService.displayed &&
-      (this.getCustomizationArgsWarningsList().length === 0));
+    let explorationTitle = this.getTitle(
+      this.stateInteractionIdService.displayed);
+    /*
+      There is a condition for the END EXPLORATION interaction
+      We have to check that the last input field for adding
+      the exploration Ids must be non-empty
+      Save Interaction button should be disable for empty input
+    */
+    if (explorationTitle === AppConstants.END_EXPLORATION) {
```

Comment:
Please move all of this end exploration interaction-related logic to a separate function.